extractor/src/main/java/rs/valence/extractor/extractors/TranslationKeys.java

```diff
@@ -0,0 +1,56 @@
+package rs.valence.extractor.extractors;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import net.minecraft.util.Language;
+import rs.valence.extractor.Main;
+
+import java.lang.reflect.Field;
+import java.util.Map;
+
+public class TranslationKeys implements Main.Extractor {
+
+    @Override
+    public String fileName() {
+        return "translation_keys.json";
+    }
+
+    @Override
+    public JsonElement extract() {
+        JsonArray translationsJson = new JsonArray();
+
+        Map<String, String> translations = extractTranslations();
```

Comment:
nit: prefer HashMap

Reply:
This specific map is initialized by the game with type `ImmutableMap<String, String>`, which can't be cast to `HashMap<String, String>`. Plus, we don't care about the implementation, just that it's a map of key-value pairs, so the generic type is good enough.